packages/circuit-ui/components/Pagination/Pagination.tsx

```diff
@@ -94,14 +94,24 @@ export const Pagination = ({
   currentPage = 1,
   totalPages,
   onChange,
-  label = 'Pagination',
-  previousLabel = 'Previous page',
-  nextLabel = 'Next page',
+  label,
+  previousLabel,
+  nextLabel,
```

Comment:
Is there a way we can keep these default values in production?

Reply:
We can set them after throwing the error I guess. Why would we want to keep the default value here, though? It should always be localized

Reply:
It _should_, but there's no guarantee that a developer sees this error since it's only thrown when visiting the affected page in development. This is unlikely though.